### ISSUE TITLE
Small mistake in 2014.md

### ISSUE THREAD

**Issue by HelixPiano** [scream314/cicada3301#9]
The text says:
'Translating each character of the word 'Interconnectedness' to its corresponding numerical value with the Gematria Primus 2013 and summing yields 772 (a prime). The song itself is 277.133 seconds long.'

I am quite sure that 772 is not a prime number.

**Comment by 95111** [comment]
You are right, 2 * 386 = 772. Do you want to do a pull request for it? If you tell me the file & line I can also do it.

**Comment by HelixPiano** [comment]
The file is 2014.md
If you search for '277.133' you will find the line.

**Comment by 95111** [comment]
I'll do a pr. Thanks for the report :)
(Don't have permission to close this issue tho, so either you or @scream314 does it.)
